app/api/v2/statistics.py

```diff
@@ -67,8 +67,16 @@ def on_get(self, req, res, contract_address=None):
             raise DataNotExistsError('contract_address: %s' % contract_address)
 
         # Get dex address
-        TokenContract = Contract.get_contract('IbetStandardTokenInterface', contract_address)
-        dex_address = TokenContract.functions.tradableExchange().call()
+        _token_contract = Contract.get_contract(
+            contract_name='IbetStandardTokenInterface',
+            address=contract_address
+        )
+        dex_address = Contract.call_function(
+            contract=_token_contract,
+            function_name="tradableExchange",
+            args=(),
+            default_returns=0
```

Comment:
default value should be `ZERO_ADDRESS` or not set, I think🤔

Reply:
That's right. I've fixed it.